ports/x11/pixman/Makefile

```diff
@@ -29,7 +29,7 @@ post-configure:
 	@echo "#define WORDS_BIGENDIAN 1" >> ${WRKSRC}/config.h
 
 do-install:
-	mkdir ${ZPREFIX}/lib
+	mkdir -p ${ZPREFIX}/lib
 	${INSTALL_LIB} ${WRKSRC}/pixman/.libs/libpixman-1.so.0.32.8 ${ZPREFIX}/lib/libpixman-1.so.0
```

Comment:
${MKDIR} should be used: https://www.freebsd.org/doc/en/books/porters-handbook/install.html